testsuite/tests/properties/invalid_struct_update/expected_concrete_syntax.lkt

```diff
@@ -0,0 +1,18 @@
+grammar None_grammar {
+    main_rule <- list*(Example(@Example))
+
+}
+
+class FooNode {
+
+    @export fun increment (kv : KV): KV = <StructUpdate>
```

Comment:
Would be cool if you added the syntax for it at the same time (in the unparser at least) if you feel up to it :)

Reply:
Good idea. I’ll try!

Reply:
Thanks!